docs/features.md

```diff
@@ -39,3 +39,23 @@ resource, Monokle will quickly show you, that you have a dead link with a warnin
 
 Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Objects" button to import all objects from
 the cluster into Monokle - [read more](cluster-integration.md)
+
+
+## Reload Folder On Startup
```

Comment:
none of these three need to be in the features files - lets move them as follows:
- move "Reload Folder On Startup" and "Navigation History" to the resource-navigation.md file
- move "Editing kustomization.." to both kustomize.md and helm.md